packages/helm-charts/oracle/templates/statefulset.yaml

```diff
@@ -43,7 +73,16 @@ spec:
           - |
             [[ $REPLICA_NAME =~ -([0-9]+)$ ]] || exit 1
             RID=${BASH_REMATCH[1]}
-            export ORACLE_KEY_PATH="/private-keys/private-key-$RID"
+
+            # Get the correct key vault name
+            AZURE_KEY_VAULTS={{- range $index, $identity := .Values.oracle.identities -}}{{ $identity.azure.keyVaultName }},{{- end }}
+            export AZURE_KEY_VAULT=`echo -n $AZURE_KEY_VAULTS | cut -d ',' -f $((RID + 1))`
+
+            # Get the correct oracle account address
+            ORACLE_ACCOUNT_ADDRESSES={{- range $index, $identity := .Values.oracle.identities -}}{{ $identity.address }},{{- end }}
+            export ORACLE_ACCOUNT_ADDRESS=`echo -n $ORACLE_ACCOUNT_ADDRESSES | cut -d ',' -f $((RID + 1))`
+
+            sleep 120
```

Comment:
I guess the `sleep` is for waiting to `aad-pod-identity` to assign the label identity, am I right?
Not sure if it can be refactored to use a reactive way using the metadata ip, but maybe I can check that.

Reply:
Oops, that was a hack so I didn't have to rebuild an image that would have a longer amount of time between reports. I'll remove- thanks!